Documentation/source/guides/public-key-pinning.rst

```diff
@@ -51,6 +55,14 @@ The Gini iOS SDK allows you to enable public key pinning with the Gini API throu
 If `kTSKEnforcePinning` is set to `false`, any SSL connection will be block even if the pinning fails. When it is enabled, it requires at least two hashes (the second one is a backup hash).
 When using `TrustKit` some messages are shown in the console log. It is possible to either specify a custom log for messages or disable it altogether (setting it as nil). To do so just use the `TrustKit.setLoggerBlock` method before initializing the `trustKitConfig`.
 
+.. warning::
+
+    The above digests serve as an example only. You should **always** create the digest yourself
+    from the Gini API's public key and use that one (see below). If you
+    received a digest from us then **always** validate it by comparing it to the digest you created
+    from the public key (see below). Failing to validate a digest may lead
```

Comment:
We should also show how to create the digest from a public key file:
```
$ cat gini.pub | openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | openssl enc -base64
```

Reply:
Yes, that would be helpful. I have done copy&paste from your guide 😋 💯